### DISCUSSION TITLE
Texture size not working

### DISCUSSION THREAD

**Discussion by karlos8** [soimy/msdf-bmfont-xml#81] [Q&A]
Hi, first of all thanks for this great tool!

I try to run the following command : msdf-bmfont --reuse -o fonts/arial.png -m 512,512 -s 15 -r 3 -p 1 -t msdf fonts/arial.ttf -f json
and get the following error. 

Can someone show me how to create a texture size 512x512? 
If I remove the -m 512,512 it uses the default settings and generate 2048x2048 texture image

Using following settings
========================================
version         : msdf-bmfont-xml v2.5.4
outputType      : json
filename        : fonts/ltmono.png
fontSize        : 15
charsetFile     : Unspecified, fallback to ASC-II
textureSize     : 512 512
texturePadding  : 1
border          : 0
distanceRange   : 3
fieldType       : msdf
roundDecimal    : 0
vector          : false
reuse           : true
smartSize       : false
pot             : false
square          : false
rot             : false
rtl             : false
fontFile        : fonts\ltmono.ttf
========================================
No valid charset file loaded, fallback to ASC-II
assert.js:391
    throw err;
    ^

AssertionError [ERR_ASSERTION]: textureSize format shall be: width,height
    at generateBMFont (C:\Users\Cru\AppData\Roaming\npm\node_modules\msdf-bmfont-xml\index.js:62:10)
    at ReadFileContext.callback (C:\Users\Cru\AppData\Roaming\npm\node_modules\msdf-bmfont-xml\cli.js:101:3)
    at FSReqCallback.readFileAfterOpen [as oncomplete] (fs.js:282:13) {
  generatedMessage: false,
  code: 'ERR_ASSERTION',
  actual: false,
  expected: true,
  operator: '=='
}

**Comment by soimy** [comment]
Sorry, can't reproduce. Can you specify your platform \ terminal \ locale ?
Option `--texture-size` shall be comma separated, but the option summary shows `width _ height`, not `width,height`

**Comment by karlos8** [comment]
Hi, 
Are you ok with this?
![details](https://user-images.githubusercontent.com/30337386/151793285-df9a988e-7f8b-43da-8b84-8c7ad54bbeea.PNG)

**Comment by soimy** [comment]
I'm OK but u r not ...

**Comment by karlos8** [comment]
What is the cause of the problem...I don't understand.

**Comment by soimy** [comment]
I suspect the cause might be your terminal locale, some locales use commas as decimal separator.

**Comment by karlos8** [comment]
Ok Thanks, I will try in a different terminal to see if the problem still occurs and I will be back : )

**Comment by karlos8** [comment]
Hello, You have right! The problem it was from VScode terminal. Thanks so much!
